feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/widget/SessionsItemDecoration.kt

```diff
@@ -44,7 +48,14 @@ class SessionsItemDecoration(
             if (position == RecyclerView.NO_POSITION) return
 
             val sessionItem = adapter.getItem(position) as SessionItem
-            val startTimeText = calcTimeText(position, view)
+            val isShowDateText = if (position > 0) {
```

Comment:
The following looks clearer.

```suggestion
            val shouldShowDateText = visibleSessionDate  && if (position > 0) {
```